src/components/layout/Sidebar.vue

```diff
@@ -5,8 +5,8 @@
         {{ category.categoryName }}
       </p>
       <ul class="menu-list">
-        <li v-for="item in category.components" exact v-bind:key="item.name">
-          <router-link :to="{ name: item.name, params: item.params }" v-if="item.name && !item.link" :aria-expanded="isExpanded(item) ? 'true' : 'false'" @click.native="toggle(item.index, category.index, item)">
+        <li v-for="item in category.components" v-bind:key="item.name">
```

Comment:
Why did you change this?

Reply:
This is wrong right now, and it results on Members menu entry being highlighted when you are on List unconfirmed users page (probably because they have same prefix /members)

Reply:
Ah yes, sounds good!